### PR TITLE
Add function to create a time range string given the start and end time.

### PR DESCRIPTION
**Pull request by amuamushu**
This function will be used to create the time range for the transcript.

### CODE REVIEW

File: src/main/webapp/timestamps.js [hunk 1]
```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * Returns {String} A time range from
+ * `startTimestampMs` to `endTimestampMs`.
+ */
+function createTimestampRange(startTimestampMs, endTimestampMs) {
```
Comment (n-wach) [inline src/main/webapp/timestamps.js:47]: `timestampRangeToString`?

File: src/main/webapp/timestamps.js [hunk 2]
```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * Returns {String} A time range from
```
Comment (n-wach) [inline src/main/webapp/timestamps.js:44]: Is putting `{String}` here recommended by the style guide? never seen it before

Comment (amuamushu) [reply src/main/webapp/timestamps.js:44]: I believe so: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

```
In methods and named functions, parameter and return types must be documented, 
except in the case of same-signature @overrides, where all types are omitted. 
```

File: src/main/webapp/timestamps.js (hunk 1 above)
Comment (amuamushu) [reply src/main/webapp/timestamps.js]: done, thanks!

File: src/main/webapp/timestamps.js [hunk 3]
```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * Returns {string} A time range from
```
Comment (n-wach) [inline src/main/webapp/timestamps.js:44]: Can you do `Returns a time range {string} from ...` to make it flow better? Or is this order required.

Comment (n-wach) [reply src/main/webapp/timestamps.js:44]: Just read that part of the styleguide again and it looks like `@return {string} A time range string created from ...` would be more typical

Comment (amuamushu) [reply src/main/webapp/timestamps.js:44]: The order I have is what I've seen in the [style guide](https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments) so I think it is better leave it as is.

File: src/main/webapp/timestamps.js [hunk 4]
```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * @returns {string} A time range from
```
Comment (billie-cat) [inline src/main/webapp/timestamps.js:44]: You do not need to use @return if you only have 1 line of comment. Please write it as:

Returns a time range string in the forma of "`startTimestampMs` - `endTimestampMs`"

Or something like that.

Comment (f-camp) [reply src/main/webapp/timestamps.js:44]: +1

Comment (amuamushu) [reply src/main/webapp/timestamps.js]: done, thanks!

**Pull request merged**